packages/frontend/src/hooks/echarts/useEcharts.ts

```diff
@@ -1211,7 +1214,13 @@ const useEcharts = () => {
             },
             dataset: {
                 id: 'lightdashResults',
-                source: getResultValues(rows, true),
+                source:
+                    validCartesianConfig?.layout?.xField === EMPTY_X_AXIS
+                        ? getResultValues(rows, true).map((s) => ({
+                              ...s,
+                              [EMPTY_X_AXIS]: ' ',
```

Comment:
what does the empty string do?

Reply:
So, I need the column to be present on the data list. The empty string is just so it doesn't show any value on the axis